Pod/Illuminator/extensions/XCUIElement.swift

```diff
@@ -45,7 +61,92 @@ extension XCUIElement {
         
         return result
     }
-    
-    
+
+    func swipeTo(target element: XCUIElement, direction: UISwipeGestureRecognizerDirection, failMessage: String, giveUpCondition: (XCUIElement, XCUIElement) -> Bool) throws {
+        repeat {
+            switch direction {
+            case UISwipeGestureRecognizerDirection.Down:
```

Comment:
I don't think I understand your question.  I need an enum of swipe directions, and this seems perfect.  Why would the swipe gestures I want to replay be different than the swipe gestures that can be recognized in the first place?